package.json

```diff
@@ -178,6 +178,7 @@
 		"@wordpress/babel-plugin-makepot": "3.3.0",
 		"@wordpress/browserslist-config": "2.6.0",
 		"@wordpress/dependency-extraction-webpack-plugin": "2.1.0",
+		"@wordpress/e2e-test-utils": "4.3.0",
```

Comment:
I wonder if this should live in the `devDependencies`  of the FSE app?

Reply:
If I recall, dev dependencies are supposed to be specified in the root package.json (see p4TIVU-9l4-wp). But things are changing so quickly around dependency management in calypso that I'm not really sure :)

Reply:
I thought that might be the case, just wanted to confirm 👍